examples/ExperimentalCases/ParallelConnectionExample/parallel_connection_example.py

```diff
@@ -0,0 +1,194 @@
+__doc__ = """Parallel connection example"""
+
+import numpy as np
+import sys
+
+# FIXME without appending sys.path make it more generic
+sys.path.append("../../../../")
```

Comment:
verify if this works, I had to modify in damping branch to make it work.

Reply:
Fixed